ocaml/typing/outcometree.mli

```diff
@@ -75,11 +75,16 @@ type out_global =
   | Ogf_global
   | Ogf_unrestricted
 
+type arg_label =
```

Comment:
Maybe just add a brief comment saying this definition avoids a dependency from oprint to asttypes? If I recall, that was the one you reported as troublesome.